sage-ios/SAGE/SAGE/SGSectionHeaderView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  SGSectionHeaderView.swift
+//  SAGE
+//
+//  Created by Erica Yin on 4/2/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class SGSectionHeaderView: UIView {
+    
+    let sectionTitle = UILabel()
+    let sectionHeight = CGFloat(26)
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.addSubview(self.sectionTitle)
+        self.setupSubviews()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        self.sectionTitle.sizeToFit()
```

Comment:
set the height equal to the view's height itself so you dont need to call center vertically. this way the text till still be centered and it wont look fuzzy cuz of float values in the positioning